Content.Shared/_Goobstation/Clothing/Systems/SharedSealableClothingSystem.cs

```diff
@@ -0,0 +1,446 @@
+            {
+                RequireCanInteract = true,
+                BreakOnMove = true,
+                BlockDuplicate = true
+            };
+            verb.Act = () => _doAfterSystem.TryStartDoAfter(doAfterArgs);
+        }*/
+
+        args.Verbs.Add(verb);
+    }
+
+    /// <summary>
+    /// Ensure actionEntity on map init
+    /// </summary>
+    private void OnControlMapInit(Entity<SealableClothingControlComponent> control, ref MapInitEvent args)
+    {
+        var (uid, comp) = control;
+        _actionContainerSystem.EnsureAction(uid, ref comp.SealActionEntity, comp.SealAction);
+    }
+
+    /* This should make as do after to start unsealing of suit with verb, but, for some reason i couldn't figure out, it ends with doAfter enumerator change exception
+     * Would be nice if some can fix this, yet unsealing will be possible only on incapacitated wearers
+    private void OnStartSealingDoAfter(Entity<SealableClothingControlComponent> control, ref StartSealingProcessDoAfterEvent args)
+    {
+        if (args.Cancelled)
+            return;
+
+        TryStartSealToggleProcess(control);
+    }*/
+
+    /// <summary>
+    /// Trying to start sealing on action. It'll notify wearer if process already started
+    /// </summary>
+    private void OnControlSealEvent(Entity<SealableClothingControlComponent> control, ref SealClothingEvent args)
+    {
+        var (uid, comp) = control;
+
+        if (!_actionBlockerSystem.CanInteract(args.Performer, null))
+            return;
+
+        if (comp.IsInProcess)
+        {
+            if (comp.IsCurrentlySealed)
+            {
+                _popupSystem.PopupClient(Loc.GetString(comp.SealedInProcessToggleFailPopup), uid, args.Performer);
+                _audioSystem.PlayPredicted(comp.FailSound, uid, args.Performer);
+            }
+            else
+            {
+                _popupSystem.PopupClient(Loc.GetString(comp.UnsealedInProcessToggleFailPopup), uid, args.Performer);
+                _audioSystem.PlayPredicted(comp.FailSound, uid, args.Performer);
+            }
+
+            return;
+        }
+
+        TryStartSealToggleProcess(control, args.Performer);
+    }
+
+    /// <summary>
+    /// Toggle seal on one part and starts same process on next part
+    /// </summary>
+    private void OnSealClothingDoAfter(Entity<SealableClothingControlComponent> control, ref SealClothingDoAfterEvent args)
+    {
+        var (uid, comp) = control;
+
+        if (args.Cancelled || args.Handled || args.Target == null)
+            return;
+
+        var part = args.Target;
+
+        if (!TryComp<SealableClothingComponent>(part, out var sealableComponet))
+            return;
+
+        sealableComponet.IsSealed = !comp.IsCurrentlySealed;
+
+        Dirty(part.Value, sealableComponet);
+
+        if (sealableComponet.IsSealed)
+            _audioSystem.PlayPvs(sealableComponet.SealUpSound, uid);
+        else
+            _audioSystem.PlayPvs(sealableComponet.SealUpSound, uid);
```

Comment:
Is this not the same thing?
```suggestion
        _audioSystem.PlayPvs(sealableComponet.SealUpSound, uid);
```